src/systems/filecoin_vm/state_tree/state_tree.go

```diff
@@ -30,6 +33,18 @@ func (st *StateTree_I) GetActorState(a addr.Address) actor.ActorState {
 	panic("TODO")
 }
 
+// Get the owner account address associated to a given miner actor.
+func (st *StateTree_I) GetMinerOwnerAddress(a addr.Address) (addr.Address, error) {
```

Comment:
Convenient as this looks, it's a huge dependency inversion to have the state tree depending on the miner actor substate. It's not obvious because the absent implementation means you didn't have to do an `import`. This method sends the wrong signal about the dependency relationships.

Please don't commit this: add a helper function (unimplemented) in the interpreter instead (which still has a circular dep problem, but at least at a less deep level). See the existing commented-out calls to `loadActorSubstate`.

Reply:
The Go dependency graph is a bit difficult to resolve here, but I've tentatively moved it to StorageMinerActor for now.